.github/workflows/trivy.yml

```diff
@@ -46,36 +52,36 @@ jobs:
           mv charmed-spark/${{ steps.artifact.outputs.base_artifact_name }} .
           sudo rockcraft.skopeo --insecure-policy copy \
             docker-archive:${{ steps.artifact.outputs.base_artifact_name }} \
-            docker-daemon:trivy/charmed-spark:test
+            docker-daemon:trivy/${{ matrix.flavour }}:test
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@0.28.0
         with:
-          image-ref: 'trivy/charmed-spark:test'
-          format: 'sarif'
-          output: 'trivy-results.sarif'
-          severity: 'MEDIUM,HIGH,CRITICAL'
+          image-ref: "trivy/${{ matrix.flavour }}:test"
+          format: "sarif"
+          output: "trivy-results.sarif"
+          severity: "MEDIUM,HIGH,CRITICAL"
 
       - name: Upload Trivy scan results to GitHub Security tab
         uses: github/codeql-action/upload-sarif@v2
```

Comment:
Can we bump to v3 ?

Reply:
Done in 7f899c7, also updated the trivy actions